core/src/Kokkos_OpenMPTargetSpace.hpp

```diff
@@ -248,7 +248,10 @@ struct DeepCopy<Kokkos::Experimental::OpenMPTargetSpace,
                                        omp_get_default_device()));
   }
   DeepCopy(const ExecutionSpace& exec, void* dst, const void* src, size_t n) {
-    exec.fence();
+    exec.fence(
+        "Kokkos::Impl::DeepCopy<OpenMPTargetSpace, OpenMPTargetSpace>: fence "
+        "before "
+        "copy");
```

Comment:
```suggestion
        "Kokkos::Impl::DeepCopy<OpenMPTargetSpace, OpenMPTargetSpace>: fence before copy");
```
I'm sure `clang-format` can do better.

Reply:
This is what clang-format leads to. I'm not going to do clang-format _and then_ manually polish the results. Other comments look good though

Reply:
Just as I expected, I get
```suggestion
        "Kokkos::Impl::DeepCopy<OpenMPTargetSpace, OpenMPTargetSpace>: fence "
        "before copy");
```
with `clang-format` after putting everything on one line. I don't care enough to not approve, though.